NAMESPACE

```diff
@@ -19,6 +19,7 @@ export(createTempEnvironment)
 export(deleteGoogleBigQueryTable)
 export(do_apriori)
 export(do_cmdscale)
+export(do_cmdscale_)
```

Comment:
Putting _ is a common way to distinguish NSE function and SE function. dplyr and tidyr also export functions with _ like group_by_, so I also exported it